src/Components/CreateImageWizard/validators.ts

```diff
@@ -103,3 +103,14 @@ export const isHostnameValid = (hostname: string) => {
     )
   );
 };
+
+export const isKernelNameValid = (kernelName: string) => {
+  if (!kernelName) {
+    return true;
+  }
+
+  return (
+    kernelName.length < 65 &&
+    /^[a-z0-9]|[a-z0-9][a-z0-9-]*[a-z0-9]$/.test(kernelName)
+  );
```

Comment:
Not completely sure about the validation pattern. Didn't find much information in blueprint reference or stages :thinking:

Reply:
What I've learned is that when a kernel is specified, DNF is actually used to install it. So it would need to follow whatever the rules for package names in DNF are. I'm sure there must be a standard for that we can find, although my Google-fu has failed me. I did find something for Ubuntu:

https://askubuntu.com/questions/1262528/what-are-usable-characters-are-for-package-names#:~:text=Package%20names%20(both%20source%20and,start%20with%20an%20alphanumeric%20character.

I think if we could find something similar for RH we would have our answer.